examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -252,8 +258,11 @@ void RegularGridDecomposition::exchangeMigratingParticles(AutoPasType &autoPasCo
   }
 }
 
-void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPasContainer) {
+void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPasContainer,
+                                                            ParticlePropertiesLibraryType &PPL) {
   std::array<double, _dimensionCount> reflSkinMin{}, reflSkinMax{};
+  auto functorLJ = autopas::LJFunctor<ParticleType, false, true, autopas::FunctorN3Modes::Newton3Off, false, false>(
```

Comment:
```suggestion
  auto functorLJ = autopas::LJFunctor<ParticleType, false, true, autopas::FunctorN3Modes::Both, false, false>(
```
Is there a reason to choose a mode different than the default?

Reply:
No - other than Newton3On is not needed. But I see no reason why Both is not fine - if N3L is accidentally applied there are no major problems.